tgui/packages/tgui/interfaces/OperatingComputer.js

```diff
@@ -0,0 +1,534 @@
+      blood_pressure_status = blood_pressure["status"];
+      blood_pressure_total = blood_pressure["total"];
+      if (blood_pressure_total <= 299) {
+        pressure_color = "red";
+      } else if (blood_pressure_total <= 414) {
+        pressure_color = "yellow";
+      } else if (blood_pressure_total <= 584) {
+        pressure_color = "green";
+      } else if (blood_pressure_total <=665) {
+        pressure_color = "yellow";
+      } else {
+        pressure_color = "red";
+      }
+    }
+  }
+
+  return (
+    <Table.Row>
+      <Table.Cell header textAlign="right" width={10}>Blood Pressure:</Table.Cell>
+      <Table.Cell width={10} color={pressure_color}>{blood_pressure_rendered} ({blood_pressure_status})</Table.Cell>
+      <Table.Cell header textAlign="right" width={10}>Blood Volume:</Table.Cell>
+      <Table.Cell width={10} color={pressure_color}>{blood_pressure_total} units</Table.Cell>
+    </Table.Row>
+  );
+};
+
+const DisplayRads = (props) => {
+  const { rad_stage, rad_dose } = props;
+  if (rad_stage > 0) {
+    return (
+      <Table.Row>
+        <Table.Cell header textAlign="right" color="yellow" width={10}>Radiation:</Table.Cell>
+        <Table.Cell width={10}>Stage {rad_stage}</Table.Cell>
+        <Table.Cell header textAlign="right" width={10}>Sieverts:</Table.Cell>
+        <Table.Cell width={10}>{rad_dose} units</Table.Cell>
+      </Table.Row>
+    );
+  }
+};
+
+const DisplayBrain = (props) => {
+  const { brain_damage_desc, brain_damage_value } = props;
+  if (brain_damage_desc !== "Okay") {
+    return (
+      <Table.Row>
+        <Table.Cell header textAlign="right" color="pink" width={10}>
+          Brain Damage:
+        </Table.Cell>
+        <Table.Cell width={10}>{brain_damage_desc}</Table.Cell>
+        <Table.Cell header textAlign="right" width={10}>Neuron Cohesion:</Table.Cell>
+        <Table.Cell>{((120-brain_damage_value)/120*100).toFixed(2)}%</Table.Cell>
+      </Table.Row>
+    );
+  }
+};
+
+const DisplayOrgans = (props, context) => {
+  const { organ_status } = props;
+  const { data } = useBackend(context);
+  if (data.occupied) {
+    return (
+      <Stack.Item width={20}>
+        <Table>
+          <Table.Row>
+            <Table.Cell header textAlign="right">Organ</Table.Cell>
+            <Table.Cell header>Status</Table.Cell>
+          </Table.Row>
+          {
+            organ_status.map((organ_bundle) => {
+              return (
+                <DisplayOrgan
+                  key={organ_bundle["organ_name"]}
+                  bundle={organ_bundle}
+                />
+              );
+            })
+          }
+        </Table>
+      </Stack.Item>
+    ); }
+};
+
+const DisplayOrgan = (props) => {
+  const { bundle } = props;
+  const organ_special = bundle["special"];
+  const organ_state = bundle["organ_state"];
+  const organ_name = bundle["organ_name"];
+
+  let font_color = "green";
+  let is_bold = false;
+
+  if (organ_state === "Missing") {
```

Comment:
use enum-ish defines dm-side (ala rad stages) and mirror that pattern into an enum here, i think?